src/lib/instance_metadata.py

```diff
@@ -73,13 +73,14 @@ async def _init_(self) -> Optional[instance_metadata]:
 
             if not all(result is None for result in results):
                 audience = jwt.decode(results[4], verify=False) if results[4] else ''
-                return instance_metadata(
+                metadata = InstanceMetadata(
```

Comment:
metadata is only logged - previously was returned - is it intentional?

Reply:
It wasn't intentional, thank you for catching this.